code/modules/surgery/organs/organ_external.dm

```diff
@@ -165,7 +165,7 @@
 	if(internal_organs && (brute_dam >= max_damage || (((sharp && brute >= 5) || brute >= 10) && prob(5))))
 		// Damage an internal organ
 		if(internal_organs && internal_organs.len)
-			var/obj/item/organ/I = pick(internal_organs)
+			var/obj/item/organ/internal/I = pick(internal_organs)
```

Comment:
Your implant surgery code means that non-organs could possibly be chosen for this - be careful